_includes/header.html

```diff
@@ -1,30 +1,82 @@
-<nav id="global-navbar" class="navbar navbar-expand-md navbar-toggleable navbar-dark bg-dark">
+<nav class="navbar navbar-expand-md noprint">
 
-  <a class="navbar-brand" href="https://www.opensuse.org/">
-    <img src="/assets/chameleon/dist/images/logo/logo-white.svg" class="d-inline-block align-top" alt="openSUSE Logo" width="48" height="30">
+  <a class="navbar-brand" href="/">
+    {% include opensuse.svg %}
+    <span class="navbar-title">{{ site.navbar_title }}</span>
   </a>
 
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#global-navbar-content" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#site-menu">
+    <svg class="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
+      <path d="M3 4h18v2H3V4zm0 7h18v2H3v-2zm0 7h18v2H3v-2z" />
+    </svg>
   </button>
 
-  <div class="collapse navbar-collapse" id="global-navbar-content">
-    <ul class="navbar-nav mr-auto">
-      <li class="nav-item ">
-        <a class="nav-link" href="/explore">Software</a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="/distributions">Download</a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="https://doc.opensuse.org/">Documentation</a>
+  <div class="collapse navbar-collapse" id="site-menu">
+    <ul class="nav navbar-nav mr-auto flex-md-shrink-0">
+      <!-- Categories Dropdown -->
+      <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="cat-menu-link" role="button" data-toggle="dropdown"
+          aria-haspopup="true" aria-expanded="false">
+          <svg class="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
+            <path
+              d="M16 20V4H4v15a1 1 0 0 0 1 1h11zm3 2H5a3 3 0 0 1-3-3V3a1 1 0 0 1 1-1h14a1 1 0 0 1 1 1v7h4v9a3 3 0 0 1-3 3zm-1-10v7a1 1 0 0 0 2 0v-7h-2zM6 6h6v6H6V6zm2 2v2h2V8H8zm-2 5h8v2H6v-2zm0 3h8v2H6v-2z" />
+          </svg>
+          Categories
+        </a>
+        <div class="dropdown-menu" aria-labelledby="cat-menu-link">
+          {% for category in site.categories %}
+          {% capture category_name %}{{ category | first }}{% endcapture %}
+          <a class="dropdown-item" href="/category/{{ category_name | slugify }}">
+            {{ category_name }}
+          </a>
+          {% endfor %}
+        </div>
       </li>
+
       <li class="nav-item">
-        <a class="nav-link" href="https://en.opensuse.org/">Wiki</a>
+        <a class="nav-link" href="{{ site.sourcecode }}#how-to-submit-a-new-post">
```

Comment:
Hm, it would actually be nice to remove that or add a note on how to contribute to the readme here